<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerCatchpointsBrowserWidgetForm</class>
 <widget class="QWidget" name="SeerCatchpointsBrowserWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="catchpointsTreeWidget">
     <property name="columnCount">
      <number>10</number>
     </property>
     <column>
      <property name="text">
       <string>Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Disposition</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enabled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>What</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Catch Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thread Groups</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Times</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Used</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="refreshCatchpointsToolButton">
       <property name="toolTip">
        <string>Refresh the list of catchpoints.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/oxygen/view-refresh.png</normaloff>:/seer/resources/HighContrast/view-refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addCatchpointToolButton">
       <property name="toolTip">
        <string>Add a new catchpoint.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/oxygen/document-new.png</normaloff>:/seer/resources/HighContrast/document-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteCatchpointsToolButton">
       <property name="toolTip">
        <string>Delete selected catchpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/oxygen/edit-delete.png</normaloff>:/seer/resources/HighContrast/edit-delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="enableCatchpointsToolButton">
       <property name="toolTip">
        <string>Enable selected catchpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/oxygen/list-add.png</normaloff>:/seer/resources/HighContrast/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="disableCatchpointsToolButton">
       <property name="toolTip">
        <string>Disable selected catchpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/oxygen/list-remove.png</normaloff>:/seer/resources/HighContrast/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
